server/app/auth/oidc/SerializedIdTokens.java

```diff
@@ -0,0 +1,51 @@
+package auth.oidc;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import javax.annotation.Nullable;
+
+/** Class that manages the mapping between a jsonb db column and a Java Map. */
```

Comment:
**suggestion (blocking):** More explanation of what this is serializing, current statement is a detail rather than an explanation for *why* this class exists

Reply:
Turns out that these constructors are not used. Removed